sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala

```diff
@@ -139,7 +140,13 @@ object ReloadWorkflow {
       )
     IO.write(entryPoint, depsFileContent.show)
 
-    Webpack.run(entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
+    customWebpackConfigFile match {
+      case Some(configFile) =>
+        Webpack.run("--config", configFile.getAbsolutePath, entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
```

Comment:
That’s great to be able to use a custom configuration file as you do, but I think that’s not good if we have to duplicate parts of the config between the “reload workflow” and the usual workflow.

I think that would be easy to share the same config, though. For instance, you could have a `webpack-module.js` file containing the code you pasted. Then we could have a `webpack.config.js` file that would import this one… That’s a bit complex but that would work…

Cool, could you add a test that does that?